p2p/src/message.rs

```diff
@@ -524,9 +522,32 @@ impl Encodable for V2NetworkMessage {
     }
 }
 
-struct HeaderDeserializationWrapper(Vec<block::Header>);
+/// A list of bitcoin block headers.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct HeadersMessage(pub Vec<block::Header>);
```

Comment:
Should this live over in the `message_blockdata` module with its `GetHeaders` counterpart?